x-pack/plugins/cases/server/client/attachments/validators.test.ts

```diff
@@ -6,10 +6,13 @@
  */
 
 import { validateFindCommentsPagination } from './validators';
+import { MAX_COMMENTS_PER_PAGE } from '../../../common/constants';
 
 const ERROR_MSG =
   'The number of documents is too high. Paginating through more than 10,000 documents is not possible.';
 
+const ERROR_MSG_PER_PAGE = `Too many comments perPage provided, The maximum allowed perPage is ${MAX_COMMENTS_PER_PAGE}.`;
```

Comment:
How about: `The provided perPage value was too high. The maximum allowed perPage value is ${...}.`